src/screens/earn/farm-item/farm-item.tsx

```diff
@@ -0,0 +1,138 @@
+import { TEZ_TOKEN_SLUG } from 'src/token/data/tokens-metadata';
+import { isDefined } from 'src/utils/is-defined';
+import { mutezToTz } from 'src/utils/tezos.util';
+
+import { FarmTokens } from '../farm-tokens/farm-tokens';
+import { useFarmItemStyles } from './farm-item.styles';
+
+interface Props {
+  farm: SingleFarmResponse;
+  lastStakeRecord: UserStakeValueInterface;
+}
+
+const FARM_PRECISION = 18;
+const DEFAULT_AMOUNT = 0;
+const DEFAULT_EXHANGE_RATE = 1;
+const DEFAULT_DECIMALS = 2;
+const REWARDS_DECIMALS = 6;
+
+export const FarmItem: FC<Props> = ({ farm, lastStakeRecord }) => {
+  const styles = useFarmItemStyles();
+  const dispatch = useDispatch();
+  const { navigate } = useNavigation();
+  const selectedAccount = useSelectedAccountSelector();
+  const tezos = useReadOnlyTezosToolkit(selectedAccount);
+
+  const rewardToken: FarmToken = {
+    symbol: farm.item.rewardToken.metadata.symbol,
+    thumbnailUri: farm.item.rewardToken.metadata.thumbnailUri
+  };
+
+  const stakeTokens = farm.item.tokens.map(token => {
+    const result: FarmToken = {
+      symbol: token.metadata.symbol,
+      thumbnailUri: token.metadata.thumbnailUri
+    };
+
+    if (token.metadata.symbol.toLowerCase() === TEZ_TOKEN_SLUG) {
+      result.iconName = IconNameEnum.TezToken;
+    }
+
+    return result;
+  });
+
+  const apr = isDefined(farm.item.apr) ? new BigNumber(farm.item.apr).toFixed(DEFAULT_DECIMALS) : '---';
+  const depositAmountAtomic = mutezToTz(
+    new BigNumber(lastStakeRecord?.depositAmountAtomic ?? DEFAULT_AMOUNT),
+    FARM_PRECISION
+  ).multipliedBy(farm.item.depositExchangeRate ?? DEFAULT_EXHANGE_RATE);
+  const claimableRewardsAtomic = mutezToTz(
+    new BigNumber(lastStakeRecord?.claimableRewards ?? DEFAULT_AMOUNT),
+    farm.item.rewardToken.metadata.decimals
+  ).multipliedBy(farm.item.earnExchangeRate ?? DEFAULT_EXHANGE_RATE);
+
+  const navigateToFarm = () =>
+    navigate(ModalsEnum.ManageFarmingPool, { id: farm.item.id, version: FarmVersionEnum.V3 });
+
+  const harvestAssetsApi = async () => {
+    if (isDefined(lastStakeRecord.lastStakeId)) {
+      const farmingContract = await tezos.wallet.at(farm.item.contractAddress);
+      const claimParams = farmingContract.methods.claim(lastStakeRecord.lastStakeId).toTransferParams();
+      const opParams: Array<ParamsWithKind> = [claimParams].map(transferParams => ({
+        ...transferParams,
+        kind: OpKind.TRANSACTION
+      }));
+
+      dispatch(
+        navigateAction(ModalsEnum.Confirmation, {
+          type: ConfirmationTypeEnum.InternalOperations,
+          opParams,
+          testID: 'CLAIM_REWARDS'
+        })
+      );
+    }
+  };
+
+  return (
+    <View style={styles.root}>
+      <View style={styles.bageContainer}>
+        {farm.item.type === PoolType.STABLESWAP && <Bage text="Stable Pool" color="#46BC94" style={styles.bage} />}
+        {new BigNumber(farm.item.vestingPeriodSeconds).isGreaterThan(DEFAULT_AMOUNT) && <Bage text="Long Term" />}
```

Comment:
All of stableswap farmings are with "Long Term" label now because the smallest period is 1 s. Is it as was planned?